### DISCUSSION TITLE
Admin commands?

### DISCUSSION THREAD

**Discussion by BjornB2** [n3d1117/chatgpt-telegram-bot#123] [Q&A] [answered]
Hi

I've added my telegram ID to the admin setting, but I don't see any other commands. Only the default 4 commands as I start typing / 
What are those admin commands? Or are they not implemented yet?

FYI, my ID is also in allowed users because I don't want * to have access. 

Björn

**Comment by AlexHTW** [comment, marked as answer]
Hi Björn,
you are correct, no special admin commands are implemented yet. The only command currently affected by the admin setting is /stats, adding openAI grant (only if you have grant left) and billing info for the admin. Sorry for the misleading readme text, counting on some admin commands in the future 😇.

You can leave your ID in both, it won't apply budgets to an admin. Any other string that is not * or an actual telegram ID should be fine, too, if you don't want any users.

**Reply by BjornB2** [reply]
Good to know! Thanks 🙏
